packages/config/src/cross-ecosystem-configs/moonbaseBeta.ts

```diff
@@ -32,6 +33,10 @@ export const moonbaseBetaRoutes = new ChainRoutes({
         .polkadotXcm()
         .transferAssetsToEcosystem()
         .X1(),
+      monitoring: MonitoringBuilder()
+        .monitorEvent()
+        .bridgeMessages()
+        .bridgeMessages(),
```

Comment:
This looks weird. Are you sure that you need it this way?

Reply:
The naming convention I'm using is `monitorEvent().[event from which we monitor source]().[event from which we monitor destination]()`

In the case of the moonbeam bridge the event triggered in both source and destination is `BridgeMessages`, so I would say it does make sense

Reply:
ok